app/controllers/items_controller.rb

```diff
@@ -2,8 +2,9 @@ class ItemsController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
   before_action :item_set, only: [:show, :edit, :update, :destroy]
   before_action :move_to_index, only: [:edit, :update, :destroy]
+  before_action :soldout_move_to_index, only: [:edit, :update, :destroy]
   def index
-    @items = Item.all.order("created_at DESC")
+    @items = Item.includes(:order).order("created_at DESC")
```

Comment:
```suggestion
    @items = Item.includes(:user).order("created_at DESC")
```
N＋1問題について考えられていて大変素晴らしいです！
今回はincludesメソッドでユーザーテーブルの情報を取得したいため、orderからuserへ変更しましょう。